src/ocean/task/TaskPool.d

```diff
@@ -175,12 +180,16 @@ class TaskPool ( TaskT : Task ) : ObjectPool!(Task)
             item.terminationHook({
                 --count;
                 if ((count == 0) && task.suspended())
+                {
                     task.resume();
+                }
             });
         }
 
         if (count > 0)
+        {
```

Comment:
>Well I made the comment for the if statement defined above and not for this one :)

Ah yeah, true. Sorry I mixed up them up.

>so better plain resume and let assert fire.

OK, I'll remove the extra `task.suspended()` check.